### PR TITLE
[RA1 README] Correct Chapter Titles

### LINKED ISSUE #2102: [RA1 README] ToC Title Errors for Ch 3 and Ch 4
The Titles of the chapters were already changed to replace NFVI with Cloud Infrastructure. Making the corrections in the ToC.

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #2102 

Changed Chapter titles in the ToC to align with titles in the document.

### CODE REVIEW

**Review by sukhdevkapur — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

File: doc/ref_arch/openstack/README.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ This is OpenStack based Reference Architecture (RA-1)
 ## Table of Contents
 * [Chapter 01 - Overview](chapters/chapter01.md)
 * [Chapter 02 - Architecture Requirements](chapters/chapter02.md)
-* [Chapter 03 - NFVI and VIM Architecture](chapters/chapter03.md)
-* [Chapter 04 - NFVI and VIM Component Level Architecture](chapters/chapter04.md)
+* [Chapter 03 - Cloud Infrastructure Architecture](chapters/chapter03.md)
+* [Chapter 04 - Cloud Infrastructure and VIM Component Level Architecture](chapters/chapter04.md)
```
Comment (ildikov) [inline doc/ref_arch/openstack/README.md:41]: Why is 'NFVI and VIM' replaced with 'Cloud Infrastructure' in Chapter 03 and only 'NFVI' and not 'NFVI and VIM' in Chapter 04?

File: doc/ref_arch/openstack/README.md [hunk 2]
```diff
@@ -37,8 +37,8 @@ This is OpenStack based Reference Architecture (RA-1)
 ## Table of Contents
 * [Chapter 01 - Overview](chapters/chapter01.md)
 * [Chapter 02 - Architecture Requirements](chapters/chapter02.md)
-* [Chapter 03 - NFVI and VIM Architecture](chapters/chapter03.md)
-* [Chapter 04 - NFVI and VIM Component Level Architecture](chapters/chapter04.md)
+* [Chapter 03 - Cloud Infrastructure Architecture](chapters/chapter03.md)
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/README.md:40]:
```suggestion
* [Chapter 03 - Cloud Infrastructure and VIM Architecture](chapters/chapter03.md)
```

File: doc/ref_arch/openstack/README.md [hunk 3]
```diff
@@ -37,10 +37,9 @@ This is OpenStack based Reference Architecture (RA-1)
 ## Table of Contents
 * [Chapter 01 - Overview](chapters/chapter01.md)
 * [Chapter 02 - Architecture Requirements](chapters/chapter02.md)
-* [Chapter 03 - NFVI and VIM Architecture](chapters/chapter03.md)
-* [Chapter 04 - NFVI and VIM Component Level Architecture](chapters/chapter04.md)
+* [Chapter 03 - Cloud Infrastructure and VIM Architecture](chapters/chapter03.md)
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/README.md:40]:
```suggestion
* [Chapter 03 - Cloud Infrastructure Architecture](chapters/chapter03.md)
```

Comment (pgoyal01) [inline doc/ref_arch/openstack/README.md:40]:
```suggestion
* [Chapter 03 - Cloud Infrastructure Architecture](chapters/chapter03.md)
```

File: doc/ref_arch/openstack/README.md (hunk 1 above)
Comment (ildikov) [reply doc/ref_arch/openstack/README.md:41]: Looks like it's just to match the titles in the document, should be resolved separately from this PR.

**Review by ildikov — APPROVED** [review]
Changes look good to me, ToC is consistent with the titles in the document.

**Review by karinesevilla — APPROVED** [review]

**Review by scottsteinbrueck — APPROVED** [review]

**Pull request merged**